esx_service/utils/auth_api.py

```diff
@@ -0,0 +1,336 @@
+import auth
```

Comment:
It was auto-generated when we did a pass of adding headers :-). Now it is copy-n-paste before push.